### ISSUE TITLE
Conflicting @emotion/react

### ISSUE THREAD

**Issue by medihack** [react-hook-form/devtools#141]
My app uses [Mantine](https://v5.mantine.dev/) component library, which uses Emotion under the hood. But it seems React Hook Form DevTools also uses Emotion, but a different version. When I add DevTools to my app, I now get the following warning on the console:

`You are loading @emotion/react when it is already loaded. Running multiple instances may cause problems. This can happen if multiple versions are used, or if multiple builds of the same version are used.`

Should I ignore it, or is there a way to make DevTools more "sandboxed"?

**Comment by bluebill1049** [comment]
What's the version difference? maybe we can upgrade to the latest if that's the issue, but it's a devtool so it's not bundle into your prod code, so i wouldn't worry too much about it.

**Comment by bluebill1049** [comment]
I think it's fine to ignore.

**Comment by medihack** [comment]
Yes, I will ignore it. But there are also some discussions in the Emotion repo itself if `@emotion/react` is better added as a peer dependency.

https://github.com/emotion-js/emotion/issues/2343#issuecomment-817265816
https://github.com/emotion-js/emotion/discussions/2795#discussioncomment-3031404

**Comment by bluebill1049** [comment]
oh i see, would you like to send a PR to adjust that? happy to re-release if that will resolve the issue.

**Comment by medihack** [comment]
Sure, it is as easy as moving the dependency line to peer dependencies. As version restriction, I would use `"@emotion/react": ">=11.1.5"`. I will open a PR.

**Comment by bluebill1049** [comment]
this caused issue: https://codesandbox.io/s/react-hook-form-get-started-ts-forked-o1ebc5?file=/src/index.tsx Will revert that commit for now.

**Comment by medihack** [comment]
Sure, I will investigate what's wrong here. Sorry for the trouble.

**Comment by bluebill1049** [comment]
not at all, thanks for contributing and helping. 🙏

**Comment by medihack** [comment]
> this caused issue: https://codesandbox.io/s/react-hook-form-get-started-ts-forked-o1ebc5?file=/src/index.tsx Will revert that commit for now.

I took another look into this. This import is a bit strange:  `import { DevTool } from "./src";`.
Shouldn't it be: `import { DevTool } from "@hookform/devtools";`?

Also, there is no `"@emotion/react"` in the dependencies of the linked CodeSandbox (but one has to, as it is a peer dependency). I wonder if it was more a problem of the CodeSandbox example.

**Comment by bluebill1049** [comment]
oops, you are right... was using a CSB reference to src folder i will revert (revert) your merge

**Comment by bluebill1049** [comment]
never mind, it's already released, I didn't revert your original commit.

**Comment by medihack** [comment]
> never mind, it's already released, I didn't revert your original commit. could you verify it's working?

It seems reverted as `@emotion/react` is still [direct dependency](https://github.com/react-hook-form/devtools/blob/1b1cdf9e860ea7822fde21882fcb51d79e250af9/package.json#L61) in master branch and not a peer dependency.

**Comment by bluebill1049** [comment]
oops, would like you to push your PR again? let's remerge it?

**Comment by afilp** [comment]
This is still a problem, using it with react-admin and we have these 2 different versions, which causes a warning on the console:

```
yarn why @emotion/react
yarn why v1.22.19
[1/4] Why do we have the module "@emotion/react"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "@emotion/react@11.11.1"
info Reasons this module exists
   - "@hookform#devtools" depends on it
   - Hoisted from "@hookform#devtools#@emotion#react"
info Disk size without dependencies: "1.75MB"
info Disk size with unique dependencies: "4.27MB"
info Disk size with transitive dependencies: "12.23MB"
info Number of shared dependencies: 44
=> Found "react-admin#@emotion/react@11.14.0"
info This module exists because "react-admin" depends on it.        
Done in 0.97s.
```
